src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -122,31 +145,37 @@ Don't rely solely on Lighthouse CI during production deployments, as you may mis
+The Web Vitals extension isn't a holistic assessment of page performance. In addition, the metrics it reports are highly dependent on your environmental setup and developers often have higher powered machines or better networks.
 
-### The performance tab in Chrome's DevTools
+### The Performance panel in Chrome DevTools
 
-[Chrome DevTools](https://developer.chrome.com/docs/devtools/) is a collection of in-browser development tools, including the [performance tab](https://developer.chrome.com/docs/devtools/#performance). The performance tab is a lab tool that profiles all page activity during startup or a recorded time period. It offers deep insight into everything it observes across dimensions such as network, rendering, painting and scripting activity, as well as a page's Core Web Vitals.
+[Chrome DevTools](https://developer.chrome.com/docs/devtools/) is a collection of in-browser development tools, including the [Performance panel](https://developer.chrome.com/docs/devtools/#performance). The Performance panel is a lab tool that profiles all page activity during page load or a recorded time period. It offers deep insight into everything it observes across dimensions such as network, rendering, painting and scripting activity, as well as a page's Core Web Vitals.
 
-#### When to use the performance tab
+#### When to use the Performance panel
 
-The performance tab should be used by developers during development to gain deep insight into page performance. It can be used with Lighthouse or other tools that export performance traces which can later be [loaded in the performance tab](https://developer.chrome.com/docs/devtools/evaluate-performance/reference/#load).
+The Performance panel should be used by developers during development to gain deep insight into page performance. This is particularly useful to debug responsiveness issues affecting FID or INP. Once a poorly responding interaction is identified and repeatable, the Performance panel can provide a wealth of data as to what is going on in the browser to help understand the issue, from main thread blocking, to JavaScript call stacks, to painting work.
 
-#### When _not_ to use the performance tab
+#### When _not_ to use the Performance panel
 
-The performance tab is a developer tool that provides lab data only. It's not a substitute for field data, and may not be easy to use for novice developers or those in non-developer roles.
+The Performance panel is a developer tool that provides lab data only. It's not a substitute for field data. It contains a wealth of debugging information but because of that it may not be easy to use for novice developers or those in non-developer roles.
```

Comment:
```suggestion
The Performance panel is a developer tool that provides lab data only. It's not a substitute for field data. It contains a lot of debugging information, but because of that, it may not be easy to use for novice developers or those in non-developer roles.
```